configure.py

```diff
@@ -643,6 +643,7 @@ def find_ninja():
     'test/perf/perf_idl',
     'test/perf/perf_vint',
```

Comment:
> number of cores:          32

I'm sure there are fewer.

Does the test run on all vcpus or just one?

Reply:
Tries also with --smp=1 and the number of iterations is the same, so it reports 32 but actually runs on 1

Reply:
Fixed in v6 ([d1490bb](https://github.com/scylladb/scylladb/commit/d1490bb7bfcdabe36ff50d88379e7edb7cbe2ddf))